addons/account/models/account_move.py

```diff
@@ -2484,11 +2484,11 @@ def _get_last_sequence_domain(self, relaxed=False):
         if not self.date or not self.journal_id:
             return "WHERE FALSE", {}
         where_string = "WHERE journal_id = %(journal_id)s AND name != '/'"
-        param = {'journal_id': self.journal_id.id}
+        param = {'journal_id': self.journal_id.id or self.journal_id._origin.id}
```

Comment:
You could also create the new move with `_origin.id` to avoid changing this :thinking: